src/lightweight_helper.js

```diff
@@ -0,0 +1,31 @@
+
+/**
+ * @param {String} path
+ * @param {String} integrity
+ */
+export const loadScript = (path, integrity) => {
+  // eslint-disable-next-line
+    if(self.importScripts) {
+    return importScripts(path);
+  }
+  return new Promise((resolve, reject) => {
+    const script = document.createElement('script');
+    script.src = path;
+    if (integrity) {
+      script.integrity = integrity;
+    }
```

Comment:
You're never passing `integrity`, so I think this can be removed

Reply:
This is kinda general function, so in the future we could use this attribute, even if it is not used now.